src/app/shared/pipes/measure.pipe.ts

```diff
@@ -0,0 +1,26 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { Observable } from "rxjs/Observable";
+
+import { Store } from '@ngrx/store';
+import * as fromRoot from '../../dataflow/reducers';
+
+import * as measureTo from '../utils';
+
+@Pipe({
+    name: 'Measure'
+})
+
+export class Measure implements PipeTransform {
+    constructor(private store: Store<fromRoot.State>) {
+    }
+
+    transform(temperature: number): number {
+        const measure: Observable<string> = this.store.select(fromRoot.getWeatherFiltersMeasure);
```

Comment:
@LeraKruchek yes, I know. However, I'll leave it as is, because I also have a couple dependencies related to measure of temperature in another components/pipes/directives.